source/thisTarget.ts

```diff
@@ -65,6 +66,11 @@ const storeTabData = once(async () => {
     return;
   }
 
+  // If the page is prerendering, wait for the change to be able to get the tab data so the frameId is correct
+  if ("prerendering" in document && Boolean(document.prerendering)) {
+    await pEvent(document, 'prerenderingchange')
```

Comment:
@fungairino you might not need the `if` statement. The docs say that it might fire immediately if the document is already rendered: https://developer.mozilla.org/en-US/docs/Web/API/Document/prerenderingchange_event#examples

Reply:
prerendering isnt a widely supported browser feature yet, so I'm leaning towards keeping the check to be more explicit. Especially since the linked example also still checks for the prerendering attribute.